### DISCUSSION TITLE
Purple Screen when login Ubuntu 20.04.3 with M1 MacBook Pro

### DISCUSSION THREAD

**Discussion by ShallyLai** [utmapp/UTM#3376] [UTM Help (macOS)]
**Describe the issue**

I followed the user guide from official website(https://mac.getutm.app/gallery/ubuntu-20-04) to install Ubuntu Arm64 on UTM, and not adding other extra settings. 
Everything goes well during installation, but after typing password and login, it appears purple screen without any desktop icon.
I search the internet for this question, but I do not find the solution of M1 Mac.
I need a step by step solution....... 

Everything goes well before login. ⬇️
<img width="1136" alt="截圖 2021-12-09 上午9 44 54" src="https://user-images.githubusercontent.com/79987875/145327242-c518cd73-66aa-457f-9b16-6aa597efe140.png">
After login, only purple screen. ⬇️
<img width="1136" alt="截圖 2021-12-09 上午9 33 15" src="https://user-images.githubusercontent.com/79987875/145327249-2b32fb64-f5d9-4763-bc0b-0824059e8b94.png">

**Configuration**
* UTM Version: 2.4.1(The latest)
* OS Version: MacOS Monterey 12.0.1 
* Apple Silicon M1 MacBook Pro
* Ubuntu Version: 20.04.3 LTS for Arm64

**Comment by conath** [comment]
Thank you for the detailed description. This seems to be an incompatibility between your Ubuntu install and UTM, so I'll move this post to the Discussion section. Issues is for technical problems with UTM.

Unfortunately I don't know how to fix your problem, I have followed the guide myself some time ago and it worked fine. Sometimes it takes longer than expected to load the desktop, especially the first time.

**Reply by ShallyLai** [reply]
Hello, Thanks for your answer. 
I remembered that I had installed Ubuntu with UTM a few months ago, and I can use it smoothly. Then, I changed to using the old version of UTM(2.1.2) to install Ubuntu. Everything goes well after login. I'm not sure, but think maybe something incompatible between Ubuntu installation and new version of UTM.
Hope this can help other who comes out this issue.

<img width="1136" alt="截圖 2021-12-10 下午9 18 03" src="https://user-images.githubusercontent.com/79987875/145580031-e6b293f7-aca7-44be-88a4-b76b31059e04.png">

**Reply by conath** [reply]
FWIW I just installed [Ubuntu 21.10](https://ubuntu.com/download/server/arm) in UTM 2.4.1 yesterday, works fine. I uploaded my VM configuration to [my website](https://chrisp.cafe/UTM/), section UTM Virtual Machine Templates.

**Reply by ShallyLai** [reply]
Unfortunately, after I installed Ubuntu21.10 with the latest UTM, my old version of Ubuntu appeared some error. So I have to kill all of them and reinstall again. Maybe I had installed other things before which caused incompatible. BTW, old version work well now. Hope I can finished my homework smoothly.
